Update readme with instructions to publish release to chrome store (#37)
short-d/short-ext: README.md

```

#### For Production Release

- **Method 1 (Automated):**

>     Once the release changes are merged into master branch, add a release tag with appropriate [semantic version](https://semver.org/).

Reviewer: ```suggestion
    Once the wanted changes are merged into master branch, [create a release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with the new [version](https://semver.org/). 
    Eg. `0.0.7`
```

- **Method 2 (Manual uploading):**
    
    - Run the following commands after replacing `x.y.z` with appropriate [semantic version](https://semver.org/), which then builds and generates
      `short-ext.zip` file in `build` directory.
        ```
        yarn
        GIT_TAG=x.y.z yarn build:production
        ```

>     - Upload the generated zip file to the Chrome Web Store.

Reviewer: ```suggestion
```
Just don't let people do it manually because we can't track when a new version is released.

```

#### For Production Release
Once the wanted changes are merged into master branch, [create a release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with the new [version](https://semver.org/).

> Eg. `0.0.7`

Reviewer: ```suggestion

Eg. 
```0.0.7```
```

```

#### For Production Release

> Once the wanted changes are merged into master branch, [create a release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with the new [version](https://semver.org/).\

Reviewer: ```suggestion
Once the wanted changes are merged into master branch, [create a release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with the new [version](https://semver.org/).
```
